frontend/packages/dev-console/src/components/import/GitImportForm.tsx

```diff
@@ -26,28 +27,42 @@ const GitImportForm: React.FC<FormikProps<FormikValues> & GitImportFormProps> =
   status,
   isSubmitting,
   dirty,
-}) => (
-  <Form onReset={handleReset} onSubmit={handleSubmit}>
-    <div className="co-m-pane__form">
-      <GitSection />
-      <AppSection project={values.project} />
-      <BuilderSection image={values.image} builderImages={builderImages} />
-      <ExpandCollapse>
-        <RouteSection route={values.route} />
-        <BuildConfigSection namespace={values.project.name} />
-        <DeploymentConfigSection namespace={values.project.name} />
-        <ScalingSection />
-        <LabelSection />
-      </ExpandCollapse>
-    </div>
-    <br />
-    <ButtonBar errorMessage={status && status.submitError} inProgress={isSubmitting}>
-      <Button disabled={!dirty || !_.isEmpty(errors)} type="submit" bsStyle="primary">
-        Create
-      </Button>
-      <Button type="reset">Cancel</Button>
-    </ButtonBar>
-  </Form>
-);
+}) => {
+  const [isExpanded, toggleExpandCollapse] = React.useState(false);
```

Comment:
Do not initialize a separate state just for the `Expandable` component here instead add this in the form state that we are already maintaining.